package_manager/dpkg.bzl

```diff
@@ -1,13 +1,24 @@
 def _dpkg_impl(repository_ctx):
   repository_ctx.file("file/BUILD", """
 package(default_visibility = ["//visibility:public"])
-exports_files(["pkg.deb"])
+deb_files = glob(["*.deb"])
+exports_files(deb_files)
 """)
 
+  for pkg in repository_ctx.attr.packages:
+    if pkg.find("+") > 0:
```

Comment:
This check is unnecessary now that the user doesn't have to do any conversion on their end.